### ISSUE TITLE
Find Graph definition for transfer learning 

### ISSUE THREAD

**Issue by bzhong2** [Hvass-Labs/TensorFlow-Tutorials#39]
I found that if I want to use new pre-trained model, I need a file which is for the graph definition. However, I cannot find this file on the google research blog where the checkpoints and source code of new models can be downloaded. How to write such a graph definition file or how to download it?  Moreover, based on the Transfer learning tutorial, how  can I let the entire network fine tuning with a slow learning rate instead of only the last layer?

**Comment by Hvass-Labs** [comment]
I am not sure I completely understand your question. This is not the official Google tutorials. My tutorials that use the Inception v3 model should automatically download the files you need.

Tutorial 10 was supposed to be about fine-tuning the entire Inception model instead of just the last layers. But the Google engineers never provided the so-called meta-graph that I need for this. I think they are currently working on a new system for doing this.
